### PR TITLE
change(locales): update en and de locale files

### PR DESCRIPTION
**Pull request by jjeroch**

### CODE REVIEW

File: cx-portal/src/assets/locales/de/main.json [hunk 1]
```diff
@@ -483,7 +483,7 @@
         }
       },
       "edcUpload": {
-        "formTitle": "Update your public key"
+        "formTitle": "Update your public key."
```
Comment (jjeroch) [inline cx-portal/src/assets/locales/de/main.json:486]: @manojava-gk what should that get used for? From my point of view we can use the "doc" section above for both the upload screens, no need to have a own section here

**Comment by manojava-gk** [comment]
@jjeroch I'm not able to checkout to this branch. My local git is not accepting some special chars in the branch name.

"daps": {
          "title": "Create DAPS Registration",
          "intro": "Far far away, behind the  word mountains, far from the countries Vokalia and Consonantia, there live the blind texts",
          "error": "The daps  registration failed. Please reverify the uploaded document"
        },

Here is the object for that error

File: cx-portal/src/assets/locales/de/main.json (hunk 1 above)
Comment (manojava-gk) [reply cx-portal/src/assets/locales/de/main.json:486]: In dev I searched for it. No where it is being used. We can remove this edcUpload object.

**Comment by jjeroch** [comment]
> @jjeroch I'm not able to checkout to this branch. My local git is not accepting some special chars in the branch name.
> 
> "daps": { "title": "Create DAPS Registration", 
> "intro": "Far far away, behind the word mountains, far from the countries Vokalia and Consonantia, there live the blind texts", 
> "error": "The daps registration failed. Please reverify the uploaded document" },
> 
> Here is the object for that error

What do you mean with that?

**Comment by jjeroch** [comment]
@manojava-gk please approve the pr

**Review by oyo — APPROVED** [review]

**Pull request merged**